Tests/SonarScanner.MSBuild.Tasks.UnitTest/GetAnalyzerSettingsTests.cs

```diff
@@ -386,6 +386,15 @@ public void ShouldMerge_OldServerVersion_ReturnsFalse(string language)
         public void ShouldMerge_NewServerVersion_ReturnsTrue(string language) =>
             CheckShouldMerge("7.4.1", language, ignoreExternalIssues: "true", expected: true);
 
+        [DataTestMethod]
+        [DataRow("cs")]
+        [DataRow("vbnet")]
+        public void ShouldMerge_NewServerVersion_InvalidSetting_NoError_ReturnsTrue(string language)
+        {
+            var logger = CheckShouldMerge("7.4", language, ignoreExternalIssues: "not a boolean value", expected: true);
+            logger.AssertWarningNotLogged($"Invalid value for 'sonar.{language}.roslyn.ignoreIssues'. Expecting 'true' or 'false'. Actual: 'not a boolean value'. External issues will not be imported.");
```

Comment:
can you maybe assert that no warning is logged rather than asserting that this specific warning is not logged?

because how can I as a reader spot a typo in the text (maybe there's the same warning but there's a typo in this long string )